### PR TITLE
Impl 956

### PR DESCRIPTION
**Pull request by Huyeng**
Custom wms info

### CODE REVIEW

File: inc/jeo-scripts/L.TileLayer.BetterWMS.js [hunk 1]
```diff
@@ -92,60 +93,92 @@ L.tileLayer.betterWms = function (url, options) {
...
+      en:  { view_detail: "View detail"},
+      km:  { view_detail: "មើលលម្អិត" },
+      my:  { view_detail: "ကြည့်ရန်အသေးစိတ်" },
+      lo:  { view_detail: "ເບິ່ງ​ລາຍ​ລະ​ອຽດ" },
+      th:  { view_detail: "ดูรายละเอียด" },
+      vi:  { view_detail: "Xem chi tiết" }
+    };
+
+    var view_detail_label = translations.en.view_detail;
+    if(detect_lang_site != "en-US"){
```
Comment (acorbi) [inline inc/jeo-scripts/L.TileLayer.BetterWMS.js]: @Huyeng Where is the **detect_lang_site** variable declared? I do not see it being declared/initialized on this function

Comment (Huyeng) [inline inc/jeo-scripts/L.TileLayer.BetterWMS.js]: The variable is declared in the jeo script

Comment (Huyeng) [inline inc/jeo-scripts/L.TileLayer.BetterWMS.js]: Should i increate the version to 2.3 or 2.2.36?

**Comment by Huyeng** [comment]
I don't want to re-define it, but if you would like to replace like this, it is ok.
if(document.documentElement.lang != "en-US"){

**Comment by acorbi** [comment]
@Huyeng See my latest commit. Defining/Using global variables across files is not a good practice and I prefer to define the variable local to the function. I am naming the variable differently **current_lang** vs **detect_lang_site** so there is no confusion.

Please merge the PR if you agree with my change or comment otherwise.

**Pull request merged**